ui/user/src/lib/actions/tooltip.svelte.ts

```diff
@@ -3,19 +3,35 @@ import popover from '$lib/actions/popover.svelte';
 interface TooltipOptions {
 	text: string;
 	disablePortal?: boolean;
-	maxWidthClass?: string;
+	classes?: string[];
+	placement?:
```

Comment:
nit: is it possible to define a type for this and reuse it in `InfoTooltop.svelte`?

```
export type TooltipPlacement =
	| 'top'
	| 'top-start'
	| 'top-end'
	| 'bottom'
	| 'bottom-start'
	| 'bottom-end'
	| 'left'
	| 'left-start'
	| 'left-end'
	| 'right'
	| 'right-start'
	| 'right-end';
```

Reply:
sweet! thanks